docs/get-started/sumo-logic-ui-new.md

```diff
@@ -0,0 +1,215 @@
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Closed Beta</span></a></p>
+
+:::sumo beta
+To request access, contact your Sumo Logic sales representative.
+:::
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Our [Quickstart](/docs/get-started/quickstart) introduced you to the process of getting data into Sumo Logic, searching and analyzing your data, and then sharing your findings with your colleagues. 
+
+This page will help you get to know the Sumo Logic user interface (UI).
+
+## Using the left navigation panel
+
+From the left-side navigation (nav) panel, you can access your dashboards library as well as our main functions and features, such as Log Search, Metrics, Infrastructure Monitoring, Application Monitoring, Cloud SIEM Enterprise, and Cloud SOAR.
+
+### Home
+
+When you first log in, you'll land on the Sumo Logic **Home** page > **Home** tab, which provides an at-a-glance view of the following:
+
+* Recently Opened Dashboards
+* Recently Run Searches
+* Recommended Dashboards 
+* Pinned Searches<br/><img src={useBaseUrl('img/get-started/dashboard-searches.png')} alt="dashboard-searches.png" />
+
+### Access dashboards and searches
+
+The left nav panel provides easy access to libraries, searches, folders, and your personal collection of dashboards. Click the icons at the top of the left-side nav panel to view:
+
+* Your **Library**, which contains:
+   * Your **Personal** dashboards and searches.
+   * Your own **Installed Apps** from the App Catalog.
+   * Dashboards and searches shared within your organization.<br/><img src={useBaseUrl('/img/get-started/library.png')} alt="library.png" width="500"/>
+* **Recent** dashboards and searches.<br/><img src={useBaseUrl('/img/get-started/recent.png')} alt="recent.png" width="350"/>
+* Your **Favorites** list of favorited dashboards and searches.<br/><img src={useBaseUrl('/img/get-started/favs.png')} alt="favs.png" width="350"/>
+
+Use the **Library** search bar to find the above quickly. **View as** mode should be set to **Me**.<br/><img src={useBaseUrl('img/get-started/library-search.png')} alt="library-search.png" width="550"/>
+
+If you're an admin, you can view the Library in [Content Administrator](#search-content-administrator-library) mode.
+
+:::tip
+Enlarge your working area by hiding the left-side nav panel. Just click the hamburger menu icon. To unhide it, click the hamburger menu icon again.<br/><img src={useBaseUrl('/img/get-started/hamburger.gif')} alt="hamburger.gif" width="350"/>
+:::
+
+## Using the top navigation bar
+
+### Access Apps, Admin settings
+
+The global toolbar (top nav bar) provides you access to (from left to right): [Help](#get-help-docs-community-and-more), [App Catalog](apps-integrations.md), [(Data) Configuration](/docs/send-data), [Administration](/docs/manage), and your user profile options ([Notifications](account-settings-preferences.md) and [Preferences](account-settings-preferences.md)).
```

Comment:
The link on "Help" is broken because there is no section "Get help: docs, community, and more".